client/src/components/core/ms-input/MsPasswordInput.vue

```diff
@@ -54,6 +61,8 @@ const hasFocus = ref(false);
 const props = defineProps<{
   label: string;
   modelValue?: string;
+  isWrong?: boolean;
```

Comment:
You don't need two variables. You can just rely on wether the errorMessage is set or not.

Reply:
Done